pkg/internal/request/span.go

```diff
@@ -63,36 +112,184 @@ type PidInfo struct {
 // REMINDER: any attribute here must be also added to the functions SpanOTELGetters,
 // SpanPromGetters and getDefinitions in pkg/internal/export/metric/definitions.go
```

Comment:
I just checked and this file doesn't exist anymore. We can either remove this coment or point to another file. Maybe @mariomac can clarify here.

Reply:
Oh, yeah. It was moved to: `pkg/export/attributes/attr_defs.go`.

Reply:
I wish Go had a Javadoc-like way of referencing functions and files.

Reply:
@rafaelroquetto would you mind changing those comments? thanks!

Reply:
@marctc sorry, totally missed it - updated.